src/compoenents/CustomerMailDetailPannel/MapDashboard.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Box, Button, Grid, Paper, Typography } from '@mui/material';
+import { ComposableMap, Geographies, Geography } from 'react-simple-maps';
+
+const geoUrl =
```

Comment:
Please add the following URL to the endpoint configuration file

Reply:
What URL?